package.json

```diff
@@ -106,6 +106,7 @@
     "dotenv": "4.0.0",
     "embed-video": "1.6.0",
     "es7-object-polyfill": "0.0.7",
+    "eventemitter3": "^4.0.7",
```

Comment:
For consistency would you pin this version?

Reply:
I think renovate should automatically add that change after merging.